default/stringtables/en.txt

```diff
@@ -3411,6 +3414,9 @@ Create Binary Save Files
 OPTIONS_USE_XML_ZLIB_SERIALIZATION
 Use Compression for XML Save Files
 
+OPTIONS_USE_MSG_ZLIB_CPR
```

Comment:
This and the `_DB_` strings are possibly being used incorrectly here. The `_DB_` version is the long / full description shown in the usage text output listing of options or as a tooltip, while the non-DB version is shown as the option name:

![option_name_and_description](https://user-images.githubusercontent.com/11386908/170960122-59c2dcb0-1a3a-447f-a101-321af9bdd1fa.png)

Here you've apparently used one for the server option and one for the client...?

Reply:
Changed `OPTIONS_DB_USE_MSG_ZLIB_CPR` to be an appropriate tooltip for `OPTIONS_USE_MSG_ZLIB_CPR` in https://github.com/freeorion/freeorion/pull/3854/commits/4ead6795ce2b51bce839955a679750840ada4831.